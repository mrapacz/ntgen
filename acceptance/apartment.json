{
    "id": "1234-1234",
    "type": "living",
    "isAvailable": true,
    "countryCode": "DE",
    "address": {
        "borough": "Dulsberg",
        "city": "Hamburg",
        "houseNumber": "2",
        "latitude": 53.587485,
        "longitude": 10.063215,
        "postalCode": "22049",
        "streetName": "Nordschleswiger Stra\u00dfe",
        "area": "Hamburg"
    },
    "_attachments": "attachments/",
    "_ts": 15828103462
}
