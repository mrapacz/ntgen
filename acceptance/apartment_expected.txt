class Address(NamedTuple):
    borough: str
    city: str
    house_number: str
    latitude: float
    longitude: float
    postal_code: str
    street_name: str
    area: str


class Apartment(NamedTuple):
    id: str
    type: str
    is_available: bool
    country_code: str
    address: Address
    attachments: str
    ts: int